values.yaml

```diff
@@ -499,6 +559,58 @@ connectInject:
   #     namespace-label: label-value
   namespaceSelector: null
 
+  # k8sAllowNamespaces is a list of k8s namespaces to allow Connect sidecar
+  # injection in. If a k8s namespace is not included or is listed in `k8sDenyNamespaces`,
+  # pods in that k8s namespace will not be injected even if they are explicitly
+  # annotated. Use ["*"] to automatically allow all k8s namespaces.
+  #
+  # For example, ["namespace1", "namespace2"] will only allow pods in the k8s
+  # namespaces `namespace1` and `namespace2` to have Connect sidecars injected
+  # and registered with Consul. All other k8s namespaces will be ignored.
+  #
+  # To deny all namespaces, set this to [].
```

Comment:
This comment should be for `k8sDenyNamespaces` I think.

Reply:
It's related to denying namespaces, but the mechanism for denying everything is to not put anything in the allow list. For that reason, it made sense to me to put it with the setting that needs to change.